src/components/Popover/Popover.module.scss

```diff
@@ -68,35 +68,35 @@
 
     // Arrows
     .isRootAtTop > .arrow {
-        top: 100%;
+        top: calc(100% - #{theme.$arrow-gap});
```

Comment:
I found out the advanced positioning is now broken. It is not related to this PR and it was already broken before, but if you could fix it here, it would be great. Otherwise let's file a separate issue for it.

<img width="741" alt="obrazek" src="https://github.com/user-attachments/assets/9b7f846f-8b4e-4a6b-8ddc-5fe3c7389528">

Reply:
This is because `transferProps` doesn't allow passing of `style`.  Wrapping the popover in a `div` like 

```jsx
{isPopoverOpen && (
  <div
    style={{
      position: strategy,
      top: y ? y : '',
      left: x ? x : '',
    }}
    ref={floating}
  >
    <Popover
      id="my-advanced-popover"
      placement={finalPlacement}
    >
      Auto-flipping Popover
    </Popover>
  </div>
)}
```
doesn't work because the wrapper `div` has dimensions of 0x0 so the popover doesn't switch sides as it should.
<img width="209" alt="Snímek obrazovky 2024-09-04 v 15 13 39" src="https://github.com/user-attachments/assets/6dc0be70-fdae-4e87-9394-36d3cd7b1b84">

Not sure what would be the best solution for this.

Reply:
OK, I'll find an issue for this. Thanks!

Reply:
Created #566.